osu.Game/Overlays/Comments/CommentsContainer.cs

```diff
@@ -0,0 +1,198 @@
+                        new CommentsHeader
+                        {
+                            Sort = { BindTarget = Sort },
+                            ShowDeleted = { BindTarget = ShowDeleted }
+                        },
+                        content = new FillFlowContainer
+                        {
+                            RelativeSizeAxes = Axes.X,
+                            AutoSizeAxes = Axes.Y,
+                            Direction = FillDirection.Vertical,
+                        },
+                        new Container
+                        {
+                            RelativeSizeAxes = Axes.X,
+                            AutoSizeAxes = Axes.Y,
+                            Children = new Drawable[]
+                            {
+                                new Box
+                                {
+                                    RelativeSizeAxes = Axes.Both,
+                                    Colour = OsuColour.Gray(0.2f)
+                                },
+                                new FillFlowContainer
+                                {
+                                    RelativeSizeAxes = Axes.X,
+                                    AutoSizeAxes = Axes.Y,
+                                    Direction = FillDirection.Vertical,
+                                    Children = new Drawable[]
+                                    {
+                                        deletedChildrenPlaceholder = new DeletedChildrenPlaceholder
+                                        {
+                                            ShowDeleted = { BindTarget = ShowDeleted }
+                                        },
+                                        new Container
+                                        {
+                                            AutoSizeAxes = Axes.Y,
+                                            RelativeSizeAxes = Axes.X,
+                                            Child = moreButton = new CommentsShowMoreButton
+                                            {
+                                                Anchor = Anchor.Centre,
+                                                Origin = Anchor.Centre,
+                                                Margin = new MarginPadding(5),
+                                                Action = () => getComments(false),
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            });
+        }
+
+        protected override void LoadComplete()
+        {
+            Sort.BindValueChanged(onSortChanged, true);
+            base.LoadComplete();
+        }
+
+        private void onSortChanged(ValueChangedEvent<CommentsSortCriteria> sort) => getComments();
+
+        private void getComments(bool initial = true)
```

Comment:
Not really a fan of this `initial` parameter. Would prefer a `clearComments()` method instead.